Introduction to Kotlin Multiplatform/07_js_canvas.md

```diff
@@ -58,9 +62,10 @@ fun renderToCanvas(canvas : HTMLCanvasElement, action: (FractalImage) -> Unit) {
 ## Using the Canvas
 
 The very last step is to call the rendering code from the `main.kt` file. 
-We add several more lines to the `src/jsMain/kotlin/main.kt` file so that 
-the `main()` function will add canvas after `img()` function call:
+Let's add several more lines to the `src/jsMain/kotlin/main.kt` file so that 
```

Comment:
Let's add a few more lines to the `src/jsMain/kotlin/main.kt` file so that 
the `main()` function adds a canvas element after the `img()` function call: